src/conversion.rs

```diff
@@ -381,78 +381,61 @@ pub trait PyTryInto<T>: Sized {
     fn try_into_exact(&self) -> Result<&T, PyDowncastError<'_>>;
 }
 
-// TryFrom implies TryInto
-impl<U> PyTryInto<U> for PyAny
-where
-    U: for<'v> PyTryFrom<'v>,
-{
-    fn try_into(&self) -> Result<&U, PyDowncastError<'_>> {
-        <U as PyTryFrom<'_>>::try_from(self)
-    }
-    fn try_into_exact(&self) -> Result<&U, PyDowncastError<'_>> {
-        U::try_from_exact(self)
-    }
-}
+mod implementations {
```

Comment:
Could you explain why you moved these into the module? Does this make the impls non-public?

Reply:
Has no functional change, it was just a precursor to #3601 where it makes it easy to slap `#[allow(deprecated)]` on it. I can revert if preferred.

Reply:
No need. I am just making sure I am not missing something here.